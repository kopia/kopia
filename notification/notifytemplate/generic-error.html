Subject: Kopia has encountered an error during {{ .EventArgs.Operation }} on {{.Hostname}}

<!doctype html>
<html>
<head>
</head>
<body>

<p><b>Operation:</b> {{ .EventArgs.OperationDetails }}</p>
<p><b>Started:</b> {{ .EventArgs.StartTimestamp | formatTime }}</p>
<p><b>Finished:</b> {{ .EventArgs.EndTimestamp | formatTime }} ({{ .EventArgs.Duration }})</p>

<p><b>Message:</b> {{ .EventArgs.ErrorMessage }}</p>

<pre>{{ .EventArgs.ErrorDetails }}</pre>

<p>Generated at {{ .EventTime | formatTime }} by <a href="https://kopia.io">Kopia {{ .KopiaBuildVersion }}</a>.</p>

</body>
</html>
