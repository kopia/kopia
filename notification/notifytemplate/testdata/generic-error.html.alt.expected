Subject: Kopia has encountered an error during Some Operation on some-host

<!doctype html>
<html>
<head>
</head>
<body>

<p><b>Operation:</b> Some Operation Details</p>
<p><b>Started:</b> Wed, 01 Jan 2020 19:04:05 PST</p>
<p><b>Finished:</b> Wed, 01 Jan 2020 19:04:06 PST (1s)</p>

<p><b>Message:</b> error message</p>

<pre>error details</pre>

<p>Generated at Wed, 01 Jan 2020 19:04:05 PST by <a href="https://kopia.io">Kopia v0-unofficial</a>.</p>

</body>
</html>
